### ISSUE TITLE
Easiest way to set up gum to use system colours by default

### ISSUE THREAD

**Issue by beauwilliams** [charmbracelet/gum#58]
I've just spun up gum to try out and with a light theme it was not very visible on the terminal. 

Now I could pass in the style for the prompt, and then write a utility function to read my system theme (light/dark) and to set prompt colour accordingly, and reuse this utility in each gum prompt.

Is there perhaps a better way to get gum to just use the colours of the shell it is running on?

**Comment by maaslalani** [comment]
Hey! You can use the base 16 colors as a color to any style flag for gum (this will use the shell theme):

For example, a prompt with your terminal's red color: 
```
gum input --prompt.foreground 1 --cursor.foreground 1
```

**Comment by mkpaz** [comment]
Thanks, that helped. But is there any reason not to use system defaults? Could you, please, at least make it usable with light themes by default as well?

**Comment by maaslalani** [comment]
Yes! I will definitely make it light theme useable by default and make an easy way to change the theme (something like `GUM_DEFAULT_COLOR`)

**Comment by maaslalani** [comment]
FYI, I improved some of the `gum choose` and commands on light mode (on latest `@main`) and adding `ENV_VARS` to customize by default further so you can make your own gum themes 🙂 

https://github.com/charmbracelet/gum/pull/92

**Comment by beauwilliams** [comment]
Excellent, perhaps with env vars we can do something like so to get the prompt to auto update on system theme change. Short example excerpted from my zsh config, reads MacOS theme setting and updates theme vars on per prompt basis using precmd()

```
refresh-theme() (
if [ "$(defaults read -g AppleInterfaceStyle 2>/dev/null)" = "Dark" ];
  then echo 'dark'
  else echo 'light'
fi;
)
export EXAMPLE_GUM_THEME_SETTING=$(refresh-theme)
precmd() { export EXAMPLE_GUM_THEME_SETTING=$(refresh-theme) }

```

**Comment by maaslalani** [comment]
@AnishDe12020 Yes, you can do `<options> | gum choose --item.foreground 1`

**Comment by maaslalani** [comment]
Fixed by https://github.com/charmbracelet/gum/pull/92 which is available in the latest `v0.4.0` release!

You can set up default colours by setting environment variables.
